src/widgets/form-login/index.ts

```diff
@@ -0,0 +1 @@
+export { FormLogin } from './ui/FormLogin';
```

Comment:
on the project we are exporting via the public interface of the UI, so just add index.ts to the ui folder with export * from './ui'